ricecooker/classes/nodes.py

```diff
@@ -217,6 +230,7 @@ def to_dict(self):
             "id": self.get_node_id().hex,
             "name": self.title,
             "thumbnail": self.thumbnail.filename if self.thumbnail else None,
+            "language" : self.language,
```

Comment:
CC is going to look for "language_id" to make it clearer that it's the id, not the language object